# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/trx/add_compile_data_path.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- a/trx/game/paths.c	2026-10-18 07:45:39.324035079 +0200
+++ b/trx/game/paths.c	2026-10-18 07:45:39.378776075 +0200
@@ -1141,10 +1141,7 @@ void GamePath_Init(const SHELL_ARGS *con
         if (env != nullptr && !String_IsEmpty(env)) {
             m_Context.trx_dir = Memory_DupStr(env);
         } else {
-            m_Context.trx_dir = Shell_GetBasePath();
-            if (m_Context.trx_dir == nullptr) {
-                m_Context.trx_dir = Memory_DupStr(".");
-            }
+            m_Context.trx_dir = Memory_DupStr("/usr/share/trx");
         }
         M_TrimTrailingSeparators(m_Context.trx_dir);
     }
